backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -67,4 +67,13 @@ export class TransactionLinkResolver {
 
     return new TransactionLink(transactionLink, new User(user))
   }
+
+  @Authorized([RIGHTS.QUERY_TRANSACTION_LINK])
+  @Query(() => TransactionLink)
+  async queryTransactionLink(@Arg('code') code: string): Promise<TransactionLink> {
+    const transactionLink = await dbTransactionLink.findOneOrFail({ code })
+    const userRepository = getCustomRepository(UserRepository)
+    const user = await userRepository.findOneOrFail({ id: transactionLink.userId })
```

Comment:
This works when `transactionLink.userId === null`?

Reply:
Can the userId of the transactionLink be null? And even when it will send an exception cause it does not find any user with id: null